playground.md

```diff
@@ -1,18 +1,21 @@
 # Playground
-Playground provides a standalone MARO running environment, it also includes examples of some predefined scenarios and quick-start tutorial.
 
-## Main service in playground
+The playground provides a standalone MARO running environment, it also includes examples of some predefined scenarios and quick-start tutorials.
+
+## Main service in the playground
+
 | Service           | Description                                                | Host                       |
 |-------------------|------------------------------------------------------------|----------------------------|
 | `Redis Commander` | Redis Web GUI.                                             | http://127.0.0.1:40009     |
-| `Read the Docs`   | Local host docs.                                           | http://127.0.0.1:40010     |
-| `Jupyter Lab`     | Jupyter lab with MARO environment, examples, notebooks.    | http://127.0.0.1:40011     |
-*(Remember change ports, if you used different ports mapping.)*
+| `Jupyter Lab`     | Jupyter lab with MARO environment, examples, notebooks.    | http://127.0.0.1:40010     |
 
-## Major materials in root folder
+*(Remember to change ports, if you use different ports mapping.)*
```

Comment:
Remember to change ports, if you use different ports mapping.
->
Remember to change ports if you use a different port mapping.

Reply:
done